src/Team.java

```diff
@@ -0,0 +1,118 @@
+import java.util.ArrayList;
+import java.util.ListIterator;
+
+public class Team {
+
+    private String name;
+    private ArrayList<Player> players;
+    private Player captain;
+    private int Target;
+    private int runs;
+    private int wickets;
+    private int ballsPlayed;
+    private int totalBalls;
+
+    public Team(String name){
```

Comment:
during introduction of design principles going ahead, such implementations from constructor need to be looked into